common/internal-types/src/tickets.rs

```diff
@@ -981,6 +981,7 @@ impl From<RedeemableTicket> for TransferableWinningTicket {
 #[cfg(test)]
 pub mod tests {
     use super::*;
+    use crate::prelude::LOWEST_POSSIBLE_WINNING_PROB;
```

Comment:
Should it not be called something like `tolerance`?

Reply:
The tolerance is computed from the lowest possible representable value, which in case of our winning probs is 1E-8.